ntoskrnl/mm/ARM3/virtual.c

```diff
@@ -710,7 +743,7 @@ MiDeleteVirtualAddresses(IN ULONG_PTR Va,
                     /* The PTE was never mapped, just nuke it here */
                     MI_ERASE_PTE(PointerPte);
                 }
-
+#if 0
```

Comment:
What is the purpose of keeping this block?

Reply:
This was the change that caused the problem. I wanted to leave it as a reminder in hopes that a developer could actually make it work without the BSOD's. Then the code could be simplified in accordance with the original commit.